src/app/components/moj/moj-pagination/moj-pagination.component.html

```diff
@@ -0,0 +1,49 @@
+<nav class="moj-pagination" aria-label="Pagination navigation">
+  <app-moj-pagination-list>
+    <app-moj-pagination-item extraClasses="moj-pagination__item--prev">
```

Comment:
Change this to this:
`<li app-moj-pagination-item class="moj-pagination__item--prev">`